.github/workflows/pr-test-e2e.yml

```diff
@@ -110,6 +109,14 @@ jobs:
         if: always()
         run: kubectl --context its1 get namespaces
 
+      - name: Dump kubeflex-controller-manager logs
+        if: always()
+        run: |
+          echo "--- Dumping logs for previous kubeflex-controller-manager container ---"
+          kubectl logs -n kubeflex-system deploy/kubeflex-controller-manager -p
```

Comment:
This is already covered. See the previous job step named "show previous logs in hosting cluster".